build_tools/azure/test_script.sh

```diff
@@ -29,7 +29,7 @@ if [[ "$COVERAGE" == "true" ]]; then
 fi
 
 if [[ -n "$CHECK_WARNINGS" ]]; then
-    TEST_CMD="$TEST_CMD -Werror::DeprecationWarning -Werror::FutureWarning"
+    TEST_CMD="$TEST_CMD -Werror::DeprecationWarning -Wignore::FutureWarning"
```

Comment:
Actually let reverse this and create a cron job that will run with check_warning